modules/tinymce/src/core/main/ts/delete/DeleteUtils.ts

```diff
@@ -9,11 +9,20 @@ import { Optional, Optionals } from '@ephox/katamari';
 import { Compare, PredicateFind, SugarElement } from '@ephox/sugar';
 
 import Editor from '../api/Editor';
+import { EditorEvent } from '../api/PublicApi';
```

Comment:
```suggestion
import { EditorEvent } from '../api/util/EventDispatcher';
```
This would have just been something done by an IDE we should be avoiding importing from `PublicApi`. We probably need to setup an ESLint rule for that.

Reply:
I fixed this one and I also found one in page up/down that should also be the internal module.